topics/admin/faqs/galaxy-update-22.05.md

```diff
@@ -14,24 +14,25 @@ Here is the recommended update procedure with ansible:
 
 1. Update to 22.01 normally
 2. Change the release to 22.05, and run the upgrade
-   1. Galaxy will probably not start correctly here, ignore it.
-   2. Run the database migration manually
+   1. Galaxy will probably not start correctly here, ignore it (even if the build fail, this if fine, just ignore).
+   2. Run the database migration manually (with the galaxy user with the venv activated)
 
       ```
-      GALAXY_CONFIG_FILE=/srv/galaxy/config/galaxy.yml sh manage_db.sh -c /srv/galaxy/config/galaxy.yml upgrade
+      GALAXY_CONFIG_FILE=/srv/galaxy/config/galaxy.yml sh /srv/galaxy/server/manage_db.sh -c /srv/galaxy/config/galaxy.yml upgrade
       ```
 
-3. Update your system's ansible, you probably need something with a major version greater than 2.
-3. Set the release to `23.0` and make other required changes. There are a lot of useful changes, but the easiest procedure is probably something like:
+3. Update your system's ansible, you probably need something with a major version of at least 2.
+4. Set the release to `23.0` and make other required changes. There are a lot of useful changes, but the easiest procedure is probably something like:
 
    1. git clone https://github.com/hexylena/git-gat/
-   2. git checkout step-4
-   3. Diff and sync (e.g. `vimdiff group_vars/galaxyservers.yml git-gat/group_vars/galaxyservers.yml`) for the main configuration files:
+   2. cd git-gat
+   3. git checkout step-4
```

Comment:
we might want to replace this with a commit, because the step-4 tag will evolve over time. might want to go back in time to 2023's version of this file to link to, hmm.

Reply:
OK I will find it.

Reply:
This is the step 4 of the first time git-gat has the release 23.0: a3324eef24445646325790489b8f572d9a7581c1

Reply:
(ah those are trees not commit IDs), yeah that's the start and I guess we made a lot of changes over prep for that course, how about `git checkout c2e7bf6d3584fbf3281fb57d8024a9189f957e0e` which should be at the end of that series of updates and hopefully captures anything we discovered during testing.